### PR TITLE
Remove last GH packages reference

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -79,14 +79,18 @@ jobs:
           java-version: 1.8
           server-username: MAVEN_USERNAME
           server-password: MAVEN_PASSWORD
+      - id: install-secret-key
```
Comment (dougwettlaufer) [inline .github/workflows/release.yml:82]: Instructions per https://gist.github.com/sualeh/ae78dc16123899d7942bc38baba5203c

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -79,14 +79,18 @@ jobs:
...
+        name: Install gpg secret key
+        run: |
+          cat <(echo -e "${{ secrets.OSSRH_GPG_SECRET_KEY }}") | gpg --batch --import
```
Comment (dimas-b) [inline .github/workflows/release.yml:85]: I suppose these keys remain installed after the build is over... Do we wipe build hosts after publishing?

**Review by olim7t — APPROVED** [review]

File: .github/workflows/release.yml (hunk 2 above)
Comment (dougwettlaufer) [reply .github/workflows/release.yml:85]: I'm not entirely sure what GH does with their infra. I'm assuming after the build is complete they tear everything down.

Comment (dougwettlaufer) [reply .github/workflows/release.yml:85]: And the number of secrets the actions are already passed I think gpg key is the least of our problems :)

**Pull request merged**